hal/armv7m/stm32/l4/stm32l4.c

```diff
@@ -151,48 +166,54 @@ void _stm32_platformInit(void)
 /* RCC (Reset and Clock Controller) */
 
 
-int _stm32_rccSetDevClock(unsigned int d, u32 state)
+static inline unsigned int stm32_rccDevClkShift(int d, int begin)
+{
+	return (unsigned int)d - (unsigned int)begin;
+}
```

Comment:
This inline function can be made more useful by adding the bit as an argument:
```suggestion
static inline unsigned int stm32_rccDevClkShift(u32 bit, int d, int begin)
{
	return bit << ((unsigned int)d - (unsigned int)begin);
}
```
This would shorten the code (fewer brackets and shifts) and make it more readable.